src/system/Table/Table.js

```diff
@@ -17,10 +17,13 @@ const Table = React.forwardRef(
 		}
 
 		return (
-			<Box sx={ { width: '100%', overflowX: 'auto' } }>
+			<Box
+				className={ classNames( 'vip-table-component', className ) }
+				sx={ { width: '100%', overflowX: 'auto' } }
+			>
 				<table
 					sx={ { width: '100%', minWidth: '1024px', borderSpacing: 0, ...sx } }
-					className={ classNames( 'vip-table-component', className ) }
```

Comment:
Why can't we keep using the same name here? Do you think this will break the PENDO configuration (existing tracking in place)? Could you use vip-table-component-wrapper at the top or anything similar?